<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBtnCategory</class>
 <widget class="QWidget" name="FormBtnCategory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>197</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="FrameBtnCategory">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>197</width>
     <height>96</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="BtnShow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть &gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="LblTax">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Налог:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LblCategory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Категория:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
